boefjes/images/boefje_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+curl -s "$1" > input.json
+OUTPUT_URL=$(python3 -c "import sys, json; print(json.loads(sys.stdin.read())['output_url'])" < input.json)
+
+# Pass the task to the boefje entrypoint and save the output, or send back that the task has failed
+python -m docker_adapter < input.json > out.json || curl -s -X POST -d '{"status": "FAILED"}' "$OUTPUT_URL"
+curl -s -X POST -f -d '@out.json' -H "Content-type: application/json" "$OUTPUT_URL"
```

Comment:
I think it would be more efficient to do everything in python than to start python twice and parse the json twice.

Reply:
The main reason for starting out this way (first versions I must say) was for one part seeing if we could leverage more generic shell tooling for this part that could also live between other adapters for other languages. But with the decision to use the `docker_adapter.py` as an adapter for other images as well now this is now a bit redundant. The other reason was not introducing requests as a dependency in the image, but now that we're installing python that's not a big extra step and otherwise `urllib` is not a scary package for just a small script doing 2 requests.

Reply:
Update: installed httpx and removed the entrypoint